### ISSUE TITLE
[a11y] [Slash] Restitution doesn't use heading

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/design-system#987]
Inside the `Restitution` component we have multiple heading : 

- one on the `HeaderRestitution` component  https://github.com/AxaFrance/design-system/blob/main/slash/react/src/Restitution/HeaderRestitution.tsx
![Image](https://github.com/user-attachments/assets/0a988422-5974-44e8-9388-13b0c4d9a435)

- one on the column or row of the restitution, inside the  `SectionRestitutionTitle` component https://github.com/AxaFrance/design-system/blob/main/slash/react/src/Restitution/SectionRestitutionTitle.tsx
![Image](https://github.com/user-attachments/assets/88e2999d-0319-42fc-a2c1-686d4d7b8ac5)

Both those headings, are not actual heading in html, there are set inside divs. We should have appropriate headings here.

**Comment by Nolann-epi** [comment]
Je m'assigne l'issue
